book/website/community-handbook/accessibility/accessibility-comms-platforms.md

```diff
@@ -0,0 +1,21 @@
+
+## Access and inclusion-related support we provide on communication platforms
+
+We aim to provide the following support and normalise the following practices within our community by doing the following places:
+
+- **Adding alt text to images**: We use alt text or "alternative text" to describe images, diagrams, or other visuals within _The Turing Way_. To learn more about how to write and use alt text for your work, please use this {ref}`guidence<accessibility-alt-text>`. 
+- **Limit embedded links and provide alternatives for accessing information**: On many platforms, embedded links may not be accessible by screen readers, or may increase cognitive overload. If you are inserting a link into a message, limit the number of embedded links within a message. On Slac, avoid embedding links within replies to messages when possible, as links posted within threads are not readable for screen readers.
+- **Limit use of emojis that disrupt narrative flow**: Do not replace words with emoljis or use too many emojis to communicate important information. Screen readers usually read one emoji at a time using the alternative text and while sometimes the purpose may be clear (i.e. ```:thumbsup:``` represents an icon of a thumbs up sign), at other times it may not be (i.e. ```:partyblobcat``` represents a multicolored cat that is rotating in a circle, and is generally used in the context of a celebration or celebratory news). More broadly, strings of emojis can produce a clunky, repetitive user experience for people trying to understand the core message.  Remember that not all people are able to access the emoji reactions option within Slack, and normalise using written affirmations as well.
+- **Cross-post information to ensure openness**: People interact with _The Turing Way_ community on many different platforms. Not everyone in the *The Turing Way* community is in the Slack workspace, or on Github, in every community call, or on every social media channel. Where possible, we aim to normalise creating issues and discussions on the Github repository or post across social media channels in order to ensure the widest reach and participation possible across the community.
+- **Be sensitive to bandwidth needs**: As a digital and remote-first project, we are committed to making sure that our resources, communication platforms, and community are accessible to the widest possible audience, despite any bandwidth needs or barriers. In order to make sure that our communication platforms are accessible in low-bandwidth environments for the community, we avoid posting data-intensive content like videos directly into the channels. Instead, we prioritise hosting the content on other platforms (i.e. YouTube) where viewers have the option to adjust the data-intensiveness of the streaming platform. We also frequently turn off video streaming in team meetings in order to adjust for participants or community members that find it difficult to tune in.
```

Comment:
Generally this is a bit confusing to read because it changes from 'do this', 'avoid this' (e.g. 'Do not replace words'), to 'we avoid', 'we aim to' (e.g. 'we avoid posting data-intensive content'). 
I appreciate there may be a subtle difference to suggesting how others should behave and modelling that behaviour but for clarity it might be worth rephrasing to be consistent.